bootstrapvz/base/manifest-schema.yml

```diff
@@ -134,6 +134,7 @@ properties:
     type: object
     properties:
       backing: {type: string}
+      logicalvolume: {type: object}
```

Comment:
For better manifest validation the volume object should be split into a `standardvolume` definition and `logicalvolume` definition, like so:
```
  volume:
    type: object
    oneOf:
      - $ref: '#/definitions/standardvolume'
      - $ref: '#/definitions/logicalvolume'
definitions:
  ...
  standardvolume:
    type: object
    properties:
      backing: {type: string}
      partitions:
        type: object
        oneOf:
          - $ref: '#/definitions/no_partitions'
          - $ref: '#/definitions/partition_table'
    required: [partitions]
    additionalProperties: false
  logicalvolume:
    type: object
    properties:
      backing: lvm
      volumegroup: {type: string}
      logicalvolume: {type: string}
      partitions:
        type: object
        oneOf:
          - $ref: '#/definitions/no_partitions'
          - $ref: '#/definitions/partition_table'
    required: [volumegroup, logicalvolume, partitions]
    additionalProperties: false
```